app/src/main/res/values/untranslated_strings.xml

```diff
@@ -97,4 +97,13 @@
   <string name="temporary_translate_to_english" translatable="false">Switch lesson to English</string>
   <!-- General test activity strings. -->
   <string name="test_activity_label" translatable="false">Test-only activity</string>
+  <string name="hinglish" translatable="false">हिन्दी</string>
+  <string name="arabic" translatable="false">العربية</string>
+  <string name="english" translatable="false">English</string>
+  <string name="hindi" translatable="false">हिन्दी</string>
+  <string name="portuguese" translatable="false">Português</string>
+  <string name="brazilian_portuguese" translatable="false">Português</string>
+  <string name="swahili" translatable="false">Kiswahili</string>
+  <string name="chinese" translatable="false">中文</string>
+  <string name="french" translatable="false">Français</string>
```

Comment:
Three things:
1. We should have a comment above these strings separating out the section in the file & explaining that these are names of languages localized in that language.
2. The name of the strings should be more contextually clear, e.g.: "hinglish_localized_language_name".
3. Please only define the language names for those defined in (excluding unspecified): https://github.com/oppia/oppia-android/blob/3b7e1a4f3b8296782beed7ff56a3fd6387b60e82/model/src/main/proto/languages.proto#L9-L33

Reply:
Updated, for chinese and french this correspond to audio languages and not OppiaLanguage. Separated the same.